apis/v1alpha2/gateway_types.go

```diff
@@ -316,8 +287,14 @@ type GatewayTLSConfig struct {
 
 	// CertificateRef is a reference to a Kubernetes object that contains a TLS
 	// certificate and private key. This certificate is used to establish a TLS
-	// handshake for requests that match the hostname of the associated listener.
-	// The referenced object MUST reside in the same namespace as Gateway.
+	// handshake for requests that match the hostname of the associated
+	// listener.
+	//
+	// When this references a resource in a different namespace than the
```

Comment:
We should probably word this in the negative so it's clear:

CertificateRefs to a different namespace from the Gateway are denied UNLESS there
is a ReferencePolicy allowing the certificate to be attached. ...

Reply:
Updated, thanks!